README.md

```diff
@@ -185,6 +185,38 @@ all zones.
   xpk-test --tpu-type=v5litepod-16 --priority=medium
   ```
 
+### Workload Debugging
+#### Collect Stack Traces
+`cloud-tpu-diagnostics` PyPI package can be used to generate stack traces for workloads running in GKE. This package dumps the Python traces when a fault such as segmentation fault, floating-point exception, or illegal operation exception occurs in the program. Additionally, it will also periodically collect stack traces to help you debug situations when the program is unresponsive. You must make the following changes in the docker image running in a Kubernetes main container.
```

Comment:
add a link to the PyPI package or github package readme for more details?

Reply:
"You must make the following changes in the docker image running in a Kubernetes main container."

I am slightly confused by the wording here, could you help me understand what the Kubernetes main container is and how the python code below relates? Thank you!

Reply:
This change must be done in the main training workload that we are running on Kubernetes.